foreman-proxy/foreman-proxy.init

```diff
@@ -58,6 +58,20 @@ stop() {
     return $RETVAL
 }
 
+logrotate() {
+    echo -n $"Rotating logs for $prog: "
+    if [ -f ${FOREMAN_PROXY_PID} ]; then
+        killproc -p ${FOREMAN_PROXY_PID} -USR1
+        RETVAL=$?
+    else
```

Comment:
I think there should be an extra `echo` after `RETVAL=$?` to ensure the output of the script is formatted correctly.